integration_tests/suite/helpers/base.py

```diff
@@ -171,7 +169,6 @@ def make_bus(cls):
             exchange_name='wazo-headers',
             exchange_type='headers',
         )
-        bus.downstream_exchange_declare('wazo-headers', 'headers', upstream)
```

Comment:
note: do not forget to add you a reminder to remove this helper once everything is merged (or not used anymore)